ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -441,28 +476,25 @@ const iconConfig = (props: IconConfigProps): IconConfig => {
       tooltip: 'Annotations',
       hasContent: true,
     },
-    runtimeConfig: {
-      id: 'runtimeConfig',
+    apps: {
+      id: 'apps',
       disabled: !!loadingError,
       faIcon: null,
       label: 'Cloud Icon',
       showIcon: () => true,
       style: { height: '22px', width: '22px' },
-      tooltip: runtimeTooltip('Cloud Analysis Environment', loadingError),
-      hasContent: true,
+      tooltip: runtimeTooltip('Applications', loadingError),
+      hasContent: true, // TODO?
     },
-    apps: {
-      id: 'apps',
+    runtimeConfig: {
```

Comment:
Should this be named something else to differentiate it from other runtime configs (non-Jupyter ones), or should we handle that when another one is added?

Reply:
That makes sense. Thank you.